test/LeanCode.IntegrationTests/TestDatabaseConfig.cs

```diff
@@ -12,7 +12,7 @@ public abstract class TestDatabaseConfig
 {
     public const string ConfigEnvName = "LeanCodeIntegrationTests__Database";
 
-    public abstract ConfigurationOverrides GetConfigurationOverrides();
+    public abstract TestConnectionString GetConfigurationOverrides();
```

Comment:
```suggestion
    public abstract TestConnectionString GetConnectionString();
```

Name doesn't feel appropriate for what it returns now.

Reply:
True, will fix